server.js

```diff
@@ -80,6 +80,84 @@ io.on('connection', function(socket){
     }); 
   });
 
+
+  //Get User list
+  socket.on('get users for create chat', function(obj){
+    //Check if a user already exists with the submitted email
```

Comment:
Update or delete this comment

Reply:
I suggest updating the code in lines 86, 138 and 147:

![image](https://user-images.githubusercontent.com/44077833/99860324-f0f60d80-2b4f-11eb-9598-c48f5eb61681.png)